src/commander/commander.c

```diff
@@ -203,25 +206,46 @@ static commander_error_t _api_set_mnemonic_passphrase_enabled(
     return COMMANDER_OK;
 }
 
-static commander_error_t _api_reboot(void)
+static commander_error_t _api_reset(void)
 {
-    if (!workflow_reboot()) {
+    if (!workflow_reset()) {
         return COMMANDER_ERR_GENERIC;
     }
     return COMMANDER_OK;
 }
 
-static commander_error_t _api_reset(void)
+static commander_error_t _api_restore_from_mnemonic(const RestoreFromMnemonicRequest* request)
 {
-    if (!workflow_reset()) {
+    if (!workflow_restore_from_mnemonic(request)) {
+        return COMMANDER_ERR_GENERIC;
+    }
+    return COMMANDER_OK;
+}
+#endif
+#if defined(FIRMWARE_BASE)
+static commander_error_t _api_display_base32(const DisplayBase32Request* request) {
+    if (!workflow_display_base32(request->msg, sizeof(request->msg))) {
         return COMMANDER_ERR_GENERIC;
     }
     return COMMANDER_OK;
 }
 
-static commander_error_t _api_restore_from_mnemonic(const RestoreFromMnemonicRequest* request)
+static commander_error_t _api_base_set_config(const BaseSetConfigRequest* request) {
```

Comment:
move all the base api's to `commander/commander_bitboxbase.c`. this works nicely if all the api messages are in on Request/Response message. check `commander/commander_eth.c` as a reference, whose main commander function is much like the one in this file, just nested one level deeper:

https://github.com/digitalbitbox/bitbox02-firmware/blob/730c5b956ec1e9a421fa76aec8353f95ff6ff9e1/src/commander/commander_eth.c#L89-L102

Reply:
ack